etterlattemaler/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/etteroppgjoer/EtteroppgjoerForhaandsvarsel.kt

```diff
@@ -0,0 +1,190 @@
+}
+
+@TemplateModelHelpers
+object EtteroppgjoerForhaandsvarsel : EtterlatteTemplate<EtteroppgjoerForhaandsvarselBrevDTO>, Hovedmal {
+    override val kode: EtterlatteBrevKode = EtterlatteBrevKode.OMS_EO_FORHAANDSVARSEL
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = EtteroppgjoerForhaandsvarselBrevDTO::class,
+        languages = languages(Language.Bokmal, Language.Nynorsk, Language.English),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Etteroppgjør Forhåndsvarsel",
+            isSensitiv = true,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VIKTIG,
+            brevtype = LetterMetadata.Brevtype.INFORMASJONSBREV,
+        ),
+    ) {
+        title {
+            textExpr(
+                Language.Bokmal to "Forhåndsvarsel om etteroppgjør av omstillingsstønad for ".expr() + data.etteroppgjoersAar.format(),
+                Language.Nynorsk to "".expr() + data.etteroppgjoersAar.format(),
+                Language.English to "".expr() + data.etteroppgjoersAar.format(),
+            )
+        }
+
+        outline {
+
+            showIf(data.bosattUtland.not() and data.norskInntekt.not()) {
+                // felles minus bosatt utland og uten norsk inntekt
+                paragraph {
+                    text(
+                        Language.Bokmal to "Hvert år, når skatteoppgjøret er ferdig, sjekker Nav inntekten din for å se om du " +
+                                "har fått utbetalt riktig beløp i omstillingsstønad året før. Omstillingsstønaden din er beregnet " +
+                                "basert på nye opplysninger fra Skatteetaten.",
+                        Language.Nynorsk to "",
+                        Language.English to "",
+                    )
+                }
+            }.orShow {
+                // hvis bosatt utland med norsk inntekt
+                paragraph {
+                    textExpr(
+                        Language.Bokmal to ("Skatteoppgjøret viser kun norsk inntekt. Siden vi ikke mottar opplysninger om utenlandsk inntekt, " +
+                                "har vi lagt til grunn det du tidligere oppga som forventet utenlandsk inntekt. Hvis disse opplysningene ikke stemmer, " +
+                                "må du sende oss dokumentasjon på din faktiske inntekt fra utlandet i ").expr() + data.etteroppgjoersAar.format() + ".",
+                        Language.Nynorsk to "".expr(),
+                        Language.English to "".expr(),
+                    )
+                }
+                paragraph {
+                    textExpr(
+                        Language.Bokmal to ("Etteroppgjør skal unnlates hvis for lite utbetalt er mindre enn 25 prosent av rettsgebyret, " +
+                                "eller hvis for mye utbetalt er mindre enn ett rettsgebyr. Per ").expr() + data.dagensDato.format() + " er ett rettsgebyr " + data.rettsgebyrBeloep.format() + " kroner.",
```

Comment:
Burde vi ta med datoen som rettsgebyret er gyldig fra? Tenker bare på dersom dette gjøres bakover i tid feks? Om vi treffer den type caser. Forsåvidt ikke kritisk i denne PR'en, men kanskje noe vi burde tenke på

Reply:
Vi har datoen tilgjengelig hvis vi har behov for å spesifisere perioder med ulikt rettsgebyr. Pr nå sendes den ikke med til pensjonsbrev